src/state/REPLState.js

```diff
@@ -94,13 +95,54 @@ class REPLState {
   static Decode(encodedState) {
     let jsonState = JSON.parse(encodedState);
     return new REPLState(
-      decodeBase64(jsonState.source),
-      decodeBase64(jsonState.plugin),
-      jsonState.configs.map((configs) => {
+      decodeBase64(jsonState.base64SourceKey),
+      decodeBase64(jsonState.base64PluginKey),
+      jsonState.configIDs.map(configs => {
```

Comment:
Just a heads up these, more likely than not, will change again to S3 links after #32 is closed